apps/aecore/src/aec_headers.erl

```diff
@@ -753,8 +837,11 @@ deserialize_micro_from_binary(<<Version:32,
                             signature = Signature,
                             txs_hash = TxsHash,
                             time = Time,
-                            version = Version},
-            {ok, populate_extra(H)};
+                            version = Version,
+                            flags = << ?MICRO_HEADER_TAG:1,
+                                       PoFTag:1,
```

Comment:
Why bother with `PoFTag`, it will be overwritten by `fix_flags` anyway?

Reply:
Good catch.
I started by trying to keep the flags correct when changing other values but added fix_flags to everything when the tests kept failing. The culprit was in aec_db. I will remove the redundant code.